src/pages/ReimbursementAccount/ExistingOwnersStep.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import lodashGet from 'lodash/get';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import {goToWithdrawalAccountSetupStep} from '../../libs/actions/BankAccounts';
+import Navigation from '../../libs/Navigation/Navigation';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import CONST from '../../CONST';
+import {Exclamation} from '../../components/Icon/Expensicons';
+import Icon from '../../components/Icon';
+import colors from '../../styles/colors';
+
+const propTypes = {
+    ...withLocalizePropTypes,
```

Comment:
I'm not sure if this is a rule or not, but don't we usually put these beneath the defined prop types?

Reply:
No rule that I know of, but that seems to happen more often than not